test/unit/vendor/lime/lime_harnessTest.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+require_once __DIR__.'/../../../bootstrap/unit.php';
+require_once __DIR__.'/tools/TestCase.php';
+require_once __DIR__.'/tools/lime_no_colorizer.php';
+
+class lime_harnessTest extends TestCase
+{
+    private function makeHarness(): lime_harness
+    {
+        $harness = new lime_harness();
+        $harness->output->colorizer = new lime_no_colorizer();
+
+        return $harness;
+    }
+
+    private function whenExecuteHarnessWithOneFileWillHaveStatusCodeAndOutput($name, $expectedOverallSucceed, $expectedOutput)
+    {
+        $this->info($name);
+
+        $harness = $this->makeHarness();
+
+        $harness->register([
+            __DIR__."/fixtures/$name.php",
```

Comment:
```suggestion
            __DIR__.'/fixtures/$name.php',
```

Reply:
The $name variable need to be interpreted.

Using single quotes change the behaviour.

Reply:
Here, I had think about passing the file path (concrete) or the name (intent).

I choose to pass the intent.

Reply:
Sorry my bad.

Reply:
The action here is to include directory "test/unit/vendor" on php-cs-fixer configuration, then execute the fixer.

Reply:
Done in 3 steps

See https://github.com/FriendsOfSymfony1/symfony1/pull/371#discussion_r1559937756